src/aliceVision/sfmDataIO/AlembicExporter.cpp

```diff
@@ -109,33 +109,35 @@ void AlembicExporter::DataImpl::addCamera(const std::string& name,
   {
     OBoolProperty(userProps, "mvg_poseLocked").set(pose->isLocked());
 
-    const Mat3& R = pose->getTransform().rotation();
-    const Vec3& center = pose->getTransform().center();
+    Eigen::Matrix4d M = Eigen::Matrix4d::Identity();
```

Comment:
Create a scope for the coord system changes and add comments to explain the reasons of this conversion.